packages/flame_forge2d/lib/forge2d_game.dart

```diff
@@ -38,4 +48,35 @@ class Forge2DGame extends FlameGame {
   Vector2 screenToFlameWorld(Vector2 position) {
     return screenToWorld(position)..y *= -1;
   }
+
+  @override
+  @mustCallSuper
+  void lifecycleStateChange(AppLifecycleState state) {
```

Comment:
Does this count as a breaking change (adding the `@mustCallSuper` decorator)?

Reply:
Good with the migration instructions, but since it is so minor breaking you don't have to add an exclamation mark, so that the more breaking changes will be more visible. :)